### ISSUE TITLE
Drop Travis for GH Actions

### ISSUE THREAD

**Issue by sdmccabe** [netsiphd/netrd#312]
<https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing>

Two bits here seem relevant:

```
macOS builds need special care and attention. We want to make sure that builders on Mac have the highest quality experience at the fastest possible speeds. Therefore, we are separating out macOS usage from other build usage and offering a distinct add-on plan that will correlate directly to your macOS usage. Purchase only the credits you need and use them until you run out.

    $15 will buy you 25 000 credits (1 minute of mac build time costs 50 credits)
    Use your credits for macOS builds only when you need to run these
    Replenish your credits as you need them
    More special build environments that fall into this category will be available soon
```

```
We love our OSS teams who choose to build using TravisCI and we fully want to support that community. However, recently we have encountered significant abuse of the intention of this offering. Abusers have been tying up our build queues and causing performance reductions for everyone. In order to bring the rules back to fair playing grounds, we are implementing some changes for our public build repositories.

    For those of you who have been building on public repositories (on travis-ci.com, with no paid subscription), we will upgrade you to our trial (free) plan with a 10K credit allotment (which allows around 1000 minutes in a Linux environment).
    You will not need to change your build definitions when you are pointed to the new plan
    When your credit allotment runs out - we’d love for you to consider which of our plans will meet your needs.
    We will be offering an allotment of OSS minutes that will be reviewed and allocated on a case by case basis. Should you want to apply for these credits please open a request with Travis CI support stating that you’d like to be considered for the OSS allotment. Please include:
        Your account name and VCS provider (like travis-ci.com/github/[your account name] )
        How many credits (build minutes) you’d like to request (should your run out of credits again you can repeat the process to request more)
    Usage will be tracked under your account information so that you can better understand how many credits/minutes are being used
```

The second part in particular seems... problematic. We might want to re-evaluate what we're doing for CI.

**Comment by leotrs** [comment]
GitHub actions, all the way. Bye bye Travis. What is the deadline for the switch over?

**Comment by sdmccabe** [comment]
Today, apparently.

Maybe we should coordinate on Slack or Zoom or something about Github Actions, since I think you have more familiarity with it than I do.

**Comment by leotrs** [comment]
wtf

**Comment by sdmccabe** [comment]
Suspended the Travis integration, will uninstall fully once the action workflow is in place.
